### PR TITLE
24w39a

### PR DESCRIPTION
**Pull request by DragonWrangler1**

### CODE REVIEW

File: mods/ITEMS/vlf_armor/register.lua [hunk 1]
```diff
@@ -237,3 +237,116 @@ minetest.register_tool("vlf_armor:elytra", {
...
 })
+
+--[[ Function to check the player's inventory for a specific item
+local function check_inventory_for_item(player)
+	local inventory = player:get_inventory()
+	local player_name = player:get_player_name()
+	-- Loop through all the lists in the player's inventory
+	for list_name, list in pairs(inventory:get_lists()) do
+		for i = 1, inventory:get_size(list_name) do
+			local stack = inventory:get_stack(list_name, i)
```
Comment (crispiebacon) [inline mods/ITEMS/vlf_armor/register.lua:248]: more commented code

File: mods/ITEMS/vlf_bundles/init.lua [hunk 2]
```diff
@@ -1,109 +1,331 @@
...
+            inventory:set_stack(from_list, from_index, stack)
+            inventory:set_stack(to_list, to_index, bundle_stack)
         end
-    end)
+    end
+end)
 end
 
+for i = 1, 64 do
+	minetest.register_alias("vlf_bundles:bundle_"..i.."", "vlf_bundles:bundle")
```
Comment (crispiebacon) [inline mods/ITEMS/vlf_bundles/init.lua:330]: Won't metadata be destroyed when aliasing ?

File: mods/ITEMS/vlf_honey/init.lua [hunk 3]
```diff
@@ -130,9 +130,20 @@ minetest.register_node("vlf_honey:honey_block", {
 		}
 	},
 	selection_box = {
-		type = "regular",
+		type = "fixed",
+		fixed = {
+			{-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
```
Comment (crispiebacon) [inline mods/ITEMS/vlf_honey/init.lua:135]: selection box can go back to regular right ?

File: mods/ITEMS/vlf_spyglass/init.lua [hunk 4]
```diff
@@ -1,12 +1,12 @@
 local S = minetest.get_translator(minetest.get_current_modname())
 
--- TODO: when < minetest 5.9 isn't supported anymore, remove this variable check and replace all occurences of [hud_elem_type_field] with type
+-- TODO: when < minetest 5.9 isn't supported anymore, remove this variable check and replace all occurrences of [hud_elem_type_field] with type
```
Comment (crispiebacon) [inline mods/ITEMS/vlf_spyglass/init.lua:3]: typo ?

File: mods/ITEMS/vlf_tools/mace.lua [hunk 5]
```diff
@@ -38,31 +38,57 @@ minetest.register_tool("vlf_tools:mace", {
 		groupcaps = {
 			snappy = {times = {1.5, 0.9, 0.4}, uses = 50, maxlevel = 3},
 		},
-		damage_groups = {fleshy = 5},
+		damage_groups = {fleshy = 0},
```
Comment (crispiebacon) [inline mods/ITEMS/vlf_tools/mace.lua:41]: any particular reason for changing damage groups to 0 ?

File: mods/ITEMS/vlf_tools/register.lua [hunk 6]
```diff
@@ -271,3 +271,140 @@ vlf_tools.register_set("netherite", {
...
+
+
+
+
+
+
+
+
+
+
```
Comment (crispiebacon) [inline mods/ITEMS/vlf_tools/register.lua:283]: lots of open room

File: mods/ITEMS/vlf_trialspawners/init.lua [hunk 7]
```diff
@@ -0,0 +1,605 @@
+--[[local S = minetest.get_translator(minetest.get_current_modname())
+
+vlf_trialspawners = {}
```
Comment (crispiebacon) [inline mods/ITEMS/vlf_trialspawners/init.lua:3]: more commented code.

File: mods/PLAYER/vlf_meshhand/init.lua [hunk 8]
```diff
@@ -41,6 +41,14 @@ local node_def = {
 		shearsy_wool = { speed = 1, level = 0, uses = 0 },
 		shearsy_cobweb = { speed = 1, level = 0, uses = 0 },
 	},
+	on_place = function(itemstack, placer, pointed_thing)
+		if minetest.get_modpath("i_have_hands") then
+		itemstack = on_place(itemstack, placer, pointed_thing)
```
Comment (crispiebacon) [inline mods/PLAYER/vlf_meshhand/init.lua:46]: I thought i_have_hands mod didn't work due to the overriding of out own hands.

**Review by crispiebacon — COMMENTED** [review]
Overall looks fairly good.

File: mods/ITEMS/vlf_bundles/init.lua (hunk 2 above)
Comment (DragonWrangler1) [reply mods/ITEMS/vlf_bundles/init.lua:330]: I have tested, and it's not.

**Pull request merged**